baseline.md

```diff
@@ -1169,42 +1172,6 @@ Used to manage authorization related settings across the company.
 | What is the end-user impact? | N/A |
 | Learn more | [Microsoft Intune and Azure Log Analytics](https://techcommunity.microsoft.com/t5/device-management-in-microsoft/microsoft-intune-and-azure-log-analytics/ba-p/463145) |
 
-## Office 365 > Exchange > AdminAuditLogConfig
-*PowerShell/Exchange/Organization/AdminAuditLogConfig*
-
-###### hidden-header
-
-### Admin Audit Log Settings
```

Comment:
why was this removed?

Reply:
We removed those configs from the baseline